test/rotations.jl

```diff
@@ -62,6 +62,12 @@ include("utils.jl")
         v = log(M, pts[1], pts[2])
         @test norm(M, pts[1], v) ≈ (angles[2] - angles[1]) * sqrt(2)
 
+        # check that exp! does not have a side effect
+        q = deeocopy(pts[1])
```

Comment:
```suggestion
        q = allocate(pts[1])
```
Let's not overuse `deepcopy`.

Reply:
No, first of all this is a test, so performance should not matter that much, second, I really want the values of `pts[1]` inside, since in `exp!(M, q, q, v)` it is both input and output for `exp!`.

Reply:
Ah, sorry, it should be `copy(M, pts[1])`, and my concern is more about style than performance (`deepcopy` won't do the right thing when copying a point for some manifolds).

Reply:
Well seen.